core/Plugin/Manager.php

```diff
@@ -70,6 +70,7 @@ public static function getInstance()
 
     // These are always activated and cannot be deactivated
     protected $pluginToAlwaysActivate = array(
+        'BulkTracking',
```

Comment:
Doesn't the plugin need to be added to the list in `global.ini.php` instead?

Reply:
It's about making sure it is always activated and not be able to be deactivated. AFAIK it is in global.ini.php already. Or not sure if you mean something else?

Reply:
Ah sure. I misread the title :see_no_evil: